runtime/oti/bcnames.h

```diff
@@ -243,18 +243,10 @@ extern J9_CDATA char * const sunJavaBCNames[];
 #define JBiincw 213
 #define JBaload0getfield 215
 #define JBnewdup 216
-#if defined(J9VM_OPT_VALHALLA_MVT)
-#define JBvload 217
-#define JBvstore 218
-#define JBvreturn 219
-#define JBvbox 220
-#define JBvunbox 221
-#define JBvaload 222
-#define JBvastore 223
+#if defined(J9VM_OPT_VALHALLA_VALUETYPES)
 #define JBvdefault 224
-#define JBvgetfield 225
```

Comment:
instead of `JBvdefault` can you name it `JBdefault` just to be consistent with the new proposal. Same for `JBvwithfield`

Reply:
In the new proposal, they renamed `vdefault` to `defaultvalue`, so I'll go with `defaultvalue` and `withfield` then?

Reply:
yes